test/test_files/nalu-nalu-panel/nalu_platform.yaml

```diff
@@ -0,0 +1,246 @@
+# -*- mode: yaml -*-
+
+Simulations:
+  - name: sim1
+    time_integrator: ti_1
+    optimizer: opt1
+
+linear_solvers:
+
+  - name: solve_scalar
```

Comment:
it would probably be best to convert these to hypre only since on some systems we don't build the tpetra solver stack

Reply:
I converted solve_scalar to hypre in both Nalu-Wind input files using the same options as solve_cont.